contracts/staking/contracts/src/stake/MixinZrxVault.sol

```diff
@@ -27,26 +27,13 @@ import "../immutable/MixinStorage.sol";
 contract MixinZrxVault is
     MixinStorage
 {
-    /// @dev Set the Zrx Vault.
-    /// @param zrxVaultAddress Address of the Zrx Vault.
-    function setZrxVault(address zrxVaultAddress)
-        external
-        onlyOwner
-    {
-        zrxVault = IZrxVault(zrxVaultAddress);
-    }
-
     /// @dev Deposits Zrx Tokens from the `owner` into the vault.
     /// @param owner of Zrx Tokens
     /// @param amount of tokens to deposit.
     function _depositFromOwnerIntoZrxVault(address owner, uint256 amount)
         internal
     {
         IZrxVault _zrxVault = zrxVault;
```

Comment:
we no longer need to cache this

Reply:
I'm probably going to just delete this mixin in a separate PR. It doesn't really do anything at the moment.